compiler/src/iree/compiler/Dialect/Stream/IR/StreamTypes.cpp

```diff
@@ -94,14 +100,16 @@ Attribute ResourceConfigAttr::parse(AsmParser &p, Type type) {
       minBufferRangeAlignment = value;
     } else if (key == "index_bits") {
       indexBits = value;
+    } else if (key == "alias_mutable_bindings") {
+      aliasMutableBindings = (bool)value;
```

Comment:
Any concerns about this int64 -> bool cast? All the other values are int64, so the code above just does this:
```c
    int64_t value = 0;
    if (failed(p.parseKeyword(&key)) || failed(p.parseEqual()) ||
        failed(p.parseInteger(value))) {
      return {};
    }
    if (key == "...") { _ = value; }
    else if (key == "...") { _ = value; }
```